setup.cfg

```diff
@@ -91,7 +91,7 @@ install_requires =
 
     # Metrics and logging
     seqeval
-    mlflow<=1.13.1 
```

Comment:
It's a big jump from 1.13 to the latest 1.24: https://github.com/mlflow/mlflow/releases Have you checked the change log already for any potential problems?

Reply:
Yep, works :-)